llmproxy/tests/unit_tests/llmproxy_client/test_llmproxy.py

```diff
@@ -4,64 +4,291 @@
     _setup_user_models,
 )
 import pytest
+import yaml
 
 import os
 
-from llmproxy.llmproxy import LLMProxy
+from llmproxy.llmproxy import (
+    LLMProxy,
+    UnsupportedModel,
+    UserConfigError,
+    LLMProxyConfigError,
+)
 
 
 CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+path_to_env_test = ".env.test"
```

Comment:
This looks like this is unused since you're already specifying the env test path each time you call the constructor